src/App.js

```diff
@@ -25,11 +55,55 @@ function App() {
           Too many horns
         </option>
       </select>
+      <label>
+        Choose the creature
+      </label>
+      <select onChange={this.handleKeywordChange}>
+        <option value="">
+          Lalala
+        </option>
+        <option value="narwhal">
+          Narwhal
+        </option>
+        <option value="rhino">
+          Rhino
+        </option>
+        <option value="unicorn">
+          Unicorn
+        </option>
+        <option value="unilego">
+          Unilego
+        </option>
+        <option value="triceratops">
+          Triceratops
+        </option>
+        <option value="markhor">
+          Markhor
+        </option>
+        <option value="mouflon">
+          Mouflon
+        </option>
+        <option value="addax">
+          Addax
+        </option>
+        <option value="chameleon">
+          Chameleon
+        </option>
+        <option value="lizard">
+          Lizard
+        </option>
+        <option value="dragon">
+          Dragon
+        </option>
+      </select>
+      </label>
+      {/* {filteredAnimals.map(item => <p>{item.title} has {item.horns} horn(s)</p>)} */}
+      
+      <ImageList animal={filteredHornedAnimals} />
```

Comment:
Very close! I didn't give the extra point, bc although it filters, you're still rendering all images.  This is due to you making two different arrays ```filteredHornedAnimals``` and ```filteredKeywordAnimals.``` 

The idea is that you would filter off of the same array, so that you don't have duplicate images.   But instead, you are making two.   Does this make sense? So you will only need one ```ImageList``` component for this lab.